packages/visx-geo/src/projections/Projection.tsx

```diff
@@ -149,7 +152,11 @@ export default function Projection<Datum extends GeoPermissibleObjects>({
 
   const path = geoPath().projection(currProjection);
 
-  if (pointRadius) path.pointRadius(pointRadius);
+  // @ts-expect-error Pending `@types/d3-geo` to add digits method (introduced in 3.1.0)
+  // @see https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/67363
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+  if (digits !== undefined) path.digits(digits);
```

Comment:
Context: https://github.com/d3/d3-geo/releases/tag/v3.1.0

Reply:
thanks for starting the thread there 🙏

Reply:
Resolved in https://github.com/airbnb/visx/pull/1767/commits/a3f86cb1217221103058e3a9fe3854d1739eb6a3, thanks to https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67384